### PR TITLE
User profile view

### PR DESCRIPTION
**Pull request by phlax**
update user profile view, makes use of new profile inclusion tags, adds team info to user profile

### CODE REVIEW

File: pootle/templates/user/profile.html [hunk 1]
```diff
@@ -115,12 +30,12 @@ <h3 class="user-position">
...
-    userData: {{ user.field_values|to_js }},
-    appRoot: '{{ user.get_absolute_url }}',
-  });
-});
+    $(function() {
+        PTL.user.init({
+            userData: {{ user.field_values|to_js }},
+            appRoot: '{{ user.get_absolute_url }}',
+        });
+    });
```
Comment (unho) [inline pootle/templates/user/profile.html]: Why the whitespace here?

Comment (phlax) [inline pootle/templates/user/profile.html]: because it should be indented from the script tag

Comment (unho) [inline pootle/templates/user/profile.html]: Not necessarily, but even having that in account in templates we indent by 2 spaces at a time and here you are using 4 spaces.

**Comment by unho** [comment]
lgtm

**Pull request merged**